### DISCUSSION TITLE
Moode Audio

### DISCUSSION THREAD

**Discussion by Alessjo99** [tteck/Proxmox#2732] [Ideas]
It would be nice to be able to install moode audio to make your amplifier shareable on the network.
At the moment the latter is only available for Raspberry.
https://moodeaudio.org/

**Comment by manuelkue** [comment]
Personally I use Snapcast for this scenario. It also allows synced multiroom audio and works flawlessly on a Debian LXC.

In my case the Debian LXC is connected to an USB DAC that has been forwarded to it by the Proxmox host.

**Reply by Alessjo99** [reply]
Ok, but snapcast don't work with bluetooth.
